test/integration/targets/collections/collection_root_user/ansible_collections/testns/testcoll/playbooks/roles/non_coll_role_to_call/tasks/main.yml

```
@@ -1,7 +1,7 @@
 - debug:
     msg: '{{ test_role_input | default("(undefined)") }}'
-  register: test_role_output
+  register: non_coll_role_to_call_test_role_output
 
 - assert:
     that:
-    - test_role_input is not defined or test_role_input == test_role_output.msg
+    - non_coll_role_to_call_test_role_output is not defined or non_coll_role_to_call_test_role_output == test_role_output.msg
```

test_role_output.msg is either "task static legacy embedded default collection" or "task dynamic default collection" at this point, and test_role_input should inherit from the parent block now (so it's "include another non-coll role") which is why tests are failing. It seems like it was testing that the inheritance wasn't happening.

I think the assertion should now be
```suggestion
    - 'test_role_input is defined and test_role_input == "include another non-coll role"'
```
Or maybe just
```suggestion
    - 'non_coll_role_to_call_test_role_output.msg == "include another non-coll role"'
```
Or maybe we should test that test_role_output.msg != non_coll_role_to_call_test_role_output.msg since they should never be the same now.